<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uTorrentConfPanel</class>
 <widget class="QWidget" name="uTorrentConfPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>105</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="browseExtraTorrents">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="appdataPathEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="browseAppdata">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="extraTorrentsPathEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>E&amp;xtra Torrents Directory:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>extraTorrentsPathEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>APPDATA Di&amp;rectory:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>appdataPathEdit</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browseAppdata</sender>
   <signal>clicked()</signal>
   <receiver>uTorrentConfPanel</receiver>
   <slot>browseAppdata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseExtraTorrents</sender>
   <signal>clicked()</signal>
   <receiver>uTorrentConfPanel</receiver>
   <slot>browseExtraTorrents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>uTorrentConfPanel</receiver>
   <slot>emitAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseAppdata()</slot>
  <slot>browseExtraTorrents()</slot>
  <slot>emitAccept()</slot>
 </slots>
</ui>
